client/payment-details/payment-method/bancontact/index.js

```diff
@@ -0,0 +1,117 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies.
+ */
+import PaymentDetailsPaymentMethodDetail from '../detail';
+
+/**
+ * Extracts and formats payment method details from a charge.
+ *
+ * @param {Object} charge The charge object.
+ * @return {Object}       A flat hash of all necessary values.
+ */
+const formatPaymentMethodDetails = ( charge ) => {
+	const { billing_details: billingDetails, payment_method: id } = charge;
+
+	const {
+		bank_name: bankName,
+		bic,
+	} = charge.payment_method_details.bancontact;
+
+	const { name, email, formatted_address: formattedAddress } = billingDetails;
+
+	return {
+		id,
+		bankName,
+		bic,
+		name,
+		email,
+		formattedAddress,
+	};
+};
+
+/**
+ * Placeholders to display while loading.
+ */
+const paymentMethodPlaceholders = {
+	bankName: 'bank name placeholder',
+	bic: 'bic placeholder',
+	id: 'id placeholder',
+	name: 'name placeholder',
+	email: 'email placeholder',
+	formattedAddress: 'address placeholder',
+	country: 'country placeholder',
+};
+
+const BancontactDetails = ( { charge = {}, isLoading } ) => {
+	const details = charge.payment_method_details
+		? formatPaymentMethodDetails( charge )
+		: paymentMethodPlaceholders;
+
+	const { id, bankName, bic, name, email, formattedAddress } = details;
+
+	// Shorthand for more readable code.
+	const Detail = PaymentDetailsPaymentMethodDetail;
+
+	return (
+		<div className="payment-method-details">
+			<div className="payment-method-details__column">
```

Comment:
I don't know if it's important or not, but you can also add `verified_name` info to this column. The returned object from a successful payment is here: https://stripe.com/docs/payments/bancontact/accept-a-payment?platform=web#bank-account-details, if you want you can skip it too.

Reply:
Cool, I see you and David added this to your methods, so I added it: https://github.com/Automattic/woocommerce-payments/pull/2650/commits/ee8fca414e4546db9d89424c3aa25da17f276e64